mopro-ios/MoproKit/Example/Tests/CircomTests.swift

```diff
@@ -0,0 +1,109 @@
+@testable import MoproKit
+import XCTest
+
```

Comment:
Do you think it makes sense to have tests for the library itself? As opposed to only for the example app. Not sure if this makes a huge difference in practice, just a thought

Reply:
Yeah I just notice that we can use `swift` to init and test the package (no need to run through xcode)
https://developer.apple.com/documentation/PackageDescription/Package
Ideally we can make the `mopro-ios` the swift package repo itself
and xcode example project inside

Reply:
I think it will be a refactor change
we can do this in another PR?

Reply:
Sure! I think we probably want to follow Cocoapod example usage, might be different from how Apple suggests for their packet structure